Source:   http://libmsn.svn.sourceforge.net/viewvc/libmsn?view=revision&revision=120
Upstream: Yes
Reason:   Fix test when built against openssl[>=1.0.0a]

--- trunk/msntest/msntest.cpp	2010/01/05 12:07:08	116
+++ trunk/msntest/msntest.cpp	2010/05/03 19:33:50	120
@@ -259,7 +259,13 @@
                     if(mySocketsSsl[i].isSSL && !mySocketsSsl[i].isConnected)
                     {
                         BIO *bio_socket_new;
-                        SSL_METHOD *meth=NULL;
+
+#if OPENSSL_VERSION_NUMBER >= 0x10000000L
+			const SSL_METHOD *meth=NULL;
+#else
+			SSL_METHOD *meth=NULL;
+#endif
+
                         meth=SSLv23_client_method();
                         SSLeay_add_ssl_algorithms();
                         mySocketsSsl[i].ctx = SSL_CTX_new(meth);
